weaver/core/drivers/corda-driver/constants.properties

```diff
@@ -1,16 +1,15 @@
 cordaReleaseGroup=net.corda
 cordaCoreReleaseGroup=net.corda
-cordaVersion=4.8.11
-cordaCoreVersion=4.8.11
-gradlePluginsVersion=5.0.17
-kotlinVersion=1.4.32
+cordaVersion=4.12
+cordaCoreVersion=4.12
+kotlinVersion=1.9.23
 junitVersion=4.12
-log4jVersion=2.17.1
-slf4jVersion=1.7.25
+log4jVersion=2.23.1
+slf4jVersion=2.0.12
 arrowVersion=0.10.4
-grpcVersion=1.28.1
-grpcKotlinVersion=0.1.3
-protobufVersion=3.11.1
-coroutinesVersion=1.3.3
-bcprovVersion=1.53
+grpcVersion=1.65.0
+grpcKotlinVersion=1.4.1
+coroutinesVersion=1.8.1
+bcprovName=bcpkix-jdk18on
```

Comment:
Should we specify this in `gradle.properties` or in the `build.gradle` itself? It looks out of place in this file.

Reply:
The thing is this name changes with version, so that's why I put it in `constants.properties`

Reply:
The point of `constants.properties` I guess is to put every variables there, so that something needs to be changed, no one needs to search it in `build.gradle`, simply updating `constants.properties` would be enough.